client/telemetry/src/lib.rs

```diff
@@ -69,6 +69,7 @@ pub use slog_scope::with_logger;
 pub use slog;
 
 mod worker;
+mod async_record;
```

Comment:
That's very OCD and nit-picky, but what about putting this *above* `mod worker;` in order to have alphabetical order?

Reply:
Thank you :heart:

Reply:
if we had rustfmt my editor would have done that automatically ;)